packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -0,0 +1,175 @@
+import '../ios/cocoapods.dart';
```

Comment:
For this and `vscode_validator.dart`, let's factor out anything that causes these dependencies into parameters for the message (e.g. `VsCodeValidator.extensionMarketplaceUrl`).  That way, we won't have dependencies on these files.

Reply:
Done.